templates/nscd.conf.erb

```diff
@@ -72,8 +72,8 @@
 	shared			hosts		yes
 	max-db-size		hosts		33554432
 
-<% unless @osfamily == 'RedHat' and @operatingsystemrelease.to_f < 6.0 -%>
-	enable-cache		services	<%= @nscd_service_cache %>
+<% unless @osfamily == 'RedHat' and @operatingsystemmajrelease == '5' -%>
```

Comment:
can you migrate this to the new facts hash? And if you want you can migrate the whole template to an epp one.